conf/eslint-all.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
```

Comment:
Please follow the same format as other files: http://eslint.org/docs/developer-guide/code-conventions#file-format